content/en/blog/_posts/2023-04-11-introducing-applyset-pruning.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: blog
+title: "Safer, more performant pruning in kubectl apply"
+date: 2023-04-11
+slug: advanced-server-side-apply
+---
+
+**Authors:** Katrina Verey (Shopify) and Justin Santa Barbara (Google)
+
+PLACEHOLDER: blog post for KEP http://git.k8s.io/enhancements/keps/sig-cli/3659-kubectl-apply-prune
```

Comment:
@justinsb assuming we do want to propose a blog post, should we do it later? If I've understood correctly, the main feature docs are due next Tuesday (Mar 21), but blog posts are due April 4.

Reply:
Due date for blog placeholders was the 8th of March - see https://github.com/kubernetes/sig-release/tree/master/releases/release-1.27